README.md

```diff
@@ -17,6 +17,26 @@ features Configuration Service provides APIs for subscriber management.
 4. 5G clients can connect & get complete configuration copy through grpc interface.
 5. 4G clients communicate with Webconsole through REST interface
 
+# UI
+
+Webconsole can optionally serve static files, which constitute the frontend part of the application.
+
+This functionality is not mandatory. The frontend files must be provided and placed in `webconsole/ui/frontend_files` before compilation, as they are not included in the webconsole codebase.
```

Comment:
I hesitated between putting the files on `ui/frontend_files` or `ui/example` . What do you think?
the files correspond to a very basic next.js application that looks like this:
![image](https://github.com/omec-project/webconsole/assets/10354025/657cd4e3-dd4a-479d-81a8-a2c77be10ffd)

Reply:
I do not have a strong opinion on this. Probably someone in your team can provide input about it.

Reply:
There is lots of new files for the frontend and the license check fails because they do not have license/copyright header. @patriciareinoso @gruyaume, any suggestion on how to proceed about this? Is it a good idea to have these files as part of the repo? should we point to users somewhere else to get the files? I was not expecting that many files to be part of the frontend. If you decide that we should keep the files here, you need to add copyright/license header to them.

Reply:
Actually we do not need all those files. That would be the minimum of a next.js application static files.
In fact, a simple html file would be served at \<ip\>:\<port\> on the browser

index.html
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Welcome</title>
</head>
<body>
    <h1>Hello, welcome</h1>
</body>
</html>
```
![image](https://github.com/omec-project/webconsole/assets/10354025/b59a1ad7-9c4a-4fcf-bd3b-a937d29f3bc5)

Reply:
I replaced all the files by just one simple html file. 
@gab-arrobo  what do you think ?

Reply:
Overall, it looks good to me. However, I also asked @thakurajayL to take a look at it.